style.css

```diff
@@ -145,42 +161,86 @@ span {
 }
 
 .tools li {
-  margin-right: 0.7rem;
-  margin-left: 0.7rem;
-  justify-content: center;
+  display: flex;
+  flex-direction: row;
+  flex-grow: 0;
+  left: 53.77%;
+  right: 19.34%;
+  position: static;
+  align-items: center;
+  margin: 0px 8px;
+  padding: 8px 12px;
   border-radius: 3px;
   background-color: #ebf0ee;
-  padding: 0.2rem 0.3rem;
-  color: rgb(51, 51, 51);
 }
 
 .proj-card button {
-  font-size: 1rem;
-  padding: 0.2rem 1.6rem;
-  background-color: #fff;
+  width: 122px;
+  height: 48px;
+  left: 551px;
+  top: calc(50% - 48px / 2 - 76.5px);
   border-radius: 4px;
```

Comment:
KIndly add hover effects on your buttons, kindly use drop shadow for your hover, the color for the drop shadow is stated in your design 
hint: `filter: drop-shadow(0 8px 16px rgb(54, 179, 127));`

Reply:
@xtrahuman  I fixed the problems you mentioned, could you plz check it be for submit any code review,
because I have just one chance

Reply:
the error still persist, maybe you do not understand me earlier
Kindly use  `html {
    font-family: "Inter", sans-serif;
  }`
remove the other code for stylings

Reply:
@xtrahuman  I changed the font family right now according your instruction  could you plz check it